components/Form.js

```diff
@@ -35,7 +35,9 @@ export default function Form() {
         placeholder="THIRD"
         required
       />
-      <StyledButton type="submit"> Generate </StyledButton>
+      <StyledButton type="submit" className="generate">
```

Comment:
you don't need to work with a CSS class here, but can use props instead, which is more consistent:

```suggestion
      <StyledButton type="submit" generate>
```